docs/bpmn-support.adoc

```diff
@@ -139,7 +139,6 @@ The event definition can be defined on the event or on the definitions.
 
 *Note*: the following events use icons derived from existing solutions
 
-* compensation: https://thenounproject.com/vityavorobyev/icon/443401/[Rewind Button ] "Icons made by https://thenounproject.com/vityavorobyev/[Viktor Vorobyev] from https://thenounproject.com/"
```

Comment:
Did you draw the icon by yourself ? If not, can you add the credit for the author ? 
Thank you

Reply:
Fully mine ;) credit to me?? 🙃